src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.tsx

```diff
@@ -97,6 +106,20 @@ const StakingWidget = ({
 
   const openRaiseObjectionDialog = useDialog(RaiseObjectionDialog);
 
+  const editor = useEditor({
```

Comment:
Why are we initialising editor here? I think that it should be moved to the objection dialog, similar to the DecisionDialog.